stats/package.json

```diff
@@ -10,5 +10,8 @@
   "license": "ISC",
   "devDependencies": {
     "fisher-yates": "^1.0.3"
+  },
+  "dependencies": {
+    "weighted": "^0.3.0"
```

Comment:
Not sure how it makes a difference in this case, but fisher-yates is already devDependency, so OK